libs/application/template-api-modules/src/lib/modules/templates/social-insurance-administration/social-insurance-administration-utils.ts

```diff
@@ -158,6 +163,62 @@ export const transformApplicationToHouseholdSupplementDTO = (
   return householdSupplementDTO
 }
 
+export const transformApplicationToPensionSupplementDTO = (
+  application: Application,
+  uploads: Attachment[],
+): ApplicationDTO => {
+  const {
+    selectedYear,
+    selectedMonth,
+    applicantPhonenumber,
+    bank,
+    bankAccountType,
+    comment,
+    iban,
+    swift,
+    bankName,
+    bankAddress,
+    currency,
+
+    applicationReason,
+  } = getPSApplicationAnswers(application.answers)
+  const { email } = getPSApplicationExternalData(application.externalData)
+
+  const householdSupplementDTO: ApplicationDTO = {
```

Comment:
```suggestion
  const pensionSupplementDTO: ApplicationDTO = {
```

Reply:
Done, Fixed in the feat/pension-suppliment branch